python/src/ops.cpp

```diff
@@ -3638,62 +3638,96 @@ void init_ops(py::module_& m) {
         )pbdoc");
   m.def(
       "atleast_1d",
-      &atleast_1d,
-      "a"_a,
+      [](const py::args& args,
+         StreamOrDevice s) -> std::variant<py::object, py::list> {
+        if (args.size() == 1) {
+          py::object result;
+          result = py::cast(atleast_1d(args[0].cast<array>(), s));
+          return result;
+        }
+
+        std::vector<array> arrays = args.cast<std::vector<array>>();
+        std::vector<array> result = atleast_1d(arrays, s);
+        py::list py_result = py::cast(result);
+
+        return py_result;
+      },
       py::pos_only(),
       py::kw_only(),
       "stream"_a = none,
       R"pbdoc(
-        atleast_1d(a: array, stream: Union[None, Stream, Device] = None) -> array
+        atleast_1d(*args: array, stream: Union[None, Stream, Device] = None) -> Union[array, List[array]]
 
-        Convert array to have at least one dimension.
+        Convert all arrays to have at least one dimension.
 
-        args:
-            a (array): Input array
+        Args:
+            args (array or list(array)): Input array or list of input arrays.
```

Comment:
```suggestion
            *args: Input arrays.
```

Based on https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Also would you mind changing the name from `args` to `arys` for consistency with NumPy?